### PR TITLE
Update people schema

### PR DESCRIPTION
**Pull request by cadecairos**
1. Add InternetHealthIssue model, create many-to-many relation with Person
2. Add Affiliation model, create many-to-many relation with Person
3. Add bio field to Person
4. Add partnership_logo image field to Person
5. Add quiote field to Person
6. Add a default image, set it as default on image and partnership_logo fields
7. Add script to copy default image to the right static directory in the docker container, and

### CODE REVIEW

File: app/networkapi/people/models.py [hunk 1]
```diff
@@ -33,16 +46,48 @@ def __str__(self):
...
+    partnership_logo = models.ImageField(
+        max_length=2048,
+        upload_to=person_partnership_logo_path,
+        default='images/shared/default.png',
+    )
+    affiliations = models.ManyToManyField(
+        Affiliation,
+        related_name='people',
     )
     teams = models.ManyToManyField(
```
Comment (gideonthomas) [inline app/networkapi/people/models.py]: I'm pretty sure affiliation was supposed to replace teams

File: scripts/prepare_container.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+cd app
+
+mkdir -p /app/media/images/shared
+
+cp /app/staticfiles/shared/images/default.png /app/media/images/shared
```
Comment (gideonthomas) [inline scripts/prepare_container.sh]: blank line

File: app/networkapi/people/models.py (hunk 1 above)
Comment (cadecairos) [inline app/networkapi/people/models.py]: @alanmoo ?

Comment (alanmoo) [inline app/networkapi/people/models.py]: Just saw this. Still need input since it's been merged?

**Pull request merged**